README.md

```diff
@@ -1,39 +1,80 @@
 # Flake schemas
 
+> [!NOTE]
+> Flake schemas are not yet supported in Nix.
+> You can track ongoing work in [this pull request][pr] against the upstream project.
+> Until that merges, you can [experiment with flake schemas](#experimenting-with-flake-schemas) using a candidate version of Nix.
+
 This [Nix flake][flakes] provides a set of schema definitions for commonly used flake output types.
 It's used by default for flakes that do not have a `schemas` output.
 
 It supports the following flake output types:
 
+* [`apps`][apps]
 * [`checks`][checks]
+* [`darwinConfigurations`][darwin]
+* [`darwinModules`][darwin]
 * [`devShells`][develop]
+* [`dockerImages`][docker]
+* [`formatter`][formatter]
+* [`homeConfigurations`][home]
+* [`homeModules`][home]
 * [`hydraJobs`][hydra]
 * [`legacyPackages`][legacy]
 * [`nixosConfigurations`][nixos]
 * [`nixosModules`][nixosmodules]
 * [`overlays`][overlays]
 * [`packages`][packages]
-* [`darwinConfigurations`][darwin]
-* [`darwinModules`][darwin]
-* [`homeConfigurations`][home]
-* [`homeModules`][home]
-* [`dockerImages`][docker]
 * `schemas`
+* [`templates`][templates]
+
+## Experimenting with flake schemas
+
+Flake schemas are not yet supported in Nix.
+To experiment with them, you can use the version of Nix from the [pull request][pr] to upstream.
+Here are some example commands, though please note that the first time you run one of these commands, you will build Nix in its entirety, which is resource instensive and could take a while:
```

Comment:
Maybe use an admonition instead so that it stands out instead of being buried inside a standard-looking sentence.

Reply:
Done